osf/utils/workflows.py

```diff
@@ -34,10 +34,30 @@ def values(cls):
     ('WITHDRAW', 'withdraw')
 ]
 
+REGISTRATION_TRIGGERS = DEFAULT_TRIGGERS + [
+    ('EMBARGO', 'embargo'),
+    ('WITHDRAW', 'withdraw'),
+    ('REQUEST_WITHDRAW', 'request_withdraw'),
+    ('REQUEST_EMBARGO', 'request_embargo'),
+    ('REQUEST_EMBARGO_TERMINATION', 'request_embargo_termination'),
+    ('TERMINATE_EMBARGO', 'terminate_embargo'),
```

Comment:
Yeah frankly I have no idea what the purpose of having these be upper/lower case is, but since we are just extending the behaviors for `DefaultTriggers` I'm hesitant to change anything that might touch that.